app/assets/stylesheets/calendar.css.scss

```diff
@@ -1,107 +1,167 @@
 table#schedule {
-    width: 100%;
-    border-collapse: collapse;
-    border-style: hidden;
-    empty-cells: show;
-    font-family: sans-serif;
+  width: 100%;
+  border-collapse: collapse;
+  border-style: hidden;
+  empty-cells: show;
+  font-family: sans-serif;
 }
 
 #schedule td {  
-    box-sizing: border-box;
-    padding: 0px;
-    height: 3em;
+  box-sizing: border-box;
+  padding: 0px;
+  height: 3em;
 }
 
 #schedule td.event-col {
-    width: 18%;
-    text-align: center;
+  width: 18%;
+  text-align: center;
 }
 
 #schedule .time-col {
-    width: 10%;
-    text-align: center;
+  width: 10%;
+  text-align: center;
 }
 
 
 #schedule td.event-col:empty:hover {
-    border: solid 2px #5C5C8A;
+  border: solid 2px #5C5C8A;
 }
 
 #schedule .event-parent {
-    position: relative;
-    width: 100%;
-    height: 100%;
-    box-sizing: border-box;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
 
-    display: table;
+  display: table;
 }
 
 #schedule .event {
-    border: outset 1px #666666;
-    background-color: #EFEFEF;
-    border-radius: 5px;
-    margin:0px;
-    padding: 0.2em;
-    font-size: x-small;
-    position: absolute;
-    top: 0px;
-    left: 0px;
-    box-sizing: border-box;
-    overflow: hidden; 
-    height: 100%;
-    width: 100%;
-    display: table-cell;
-    z-index: 10;
+  border: outset 1px #666666;
+  background-color: #EFEFEF;
+  border-radius: 5px;
+  margin:0px;

```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`